apis/python/src/tiledbsoma/_tdb_handles.py

```diff
@@ -809,21 +810,33 @@ def _write(self) -> None:
             # There were no changes (e.g., it's a read handle).  Do nothing.
             return
         # Only try to get the writer if there are changes to be made.
+
+        errors: MutableSequence[Exception] = []
```

Comment:
just a style comment:  if it is coded as a list (ie., `= []`), why declare a looser type.  Better just to define the type as `list[Exception]` and take advantage of stronger type checking